indexer/src/test/java/org/pdxfinder/commandline/FinderCommandLineTest.java

```diff
@@ -60,22 +63,44 @@ public void setUp() {
     }
 
     @Test public void load_givenTransform_When_exportAllisCalled_Then_callsTransformer() throws IOException {
```

Comment:
Remove `load` from method name. This comes from other tests that use the naming convention `methodNameUnderTest_givenCondition_thenBehavesCertainWay`. It's a good time to review the names for the other tests in this file too.